package.json

```diff
@@ -4,16 +4,15 @@
   "description": "A shared set of design and code elements for creating accessible and consistent websites",
   "repository": "https://github.com/CMSgov/design-system",
   "scripts": {
+    "start": "NODE_ENV=development yarn cmsds start ./packages/design-system ./packages/design-system-docs",
     "build": "NODE_ENV=production yarn cmsds build-docs ./packages/design-system ./packages/design-system-docs",
-    "bump": "lerna publish --skip-git --skip-npm",
-    "precommit": "lint-staged",
+    "release": "./prepublish.sh && yarn lerna publish from-git",
```

Comment:
@line47 @nichia This probably the largest change to our process in this PR. I replaced the publish script with `lerna publish` (as thats basically all it was doing anyway), and then I created a new npm script called "release". This new script will do everything to publish a new npm version all in one script. I figured this would simplify and automate our release process a lot more, at the expense of having less control.

Even though our release process includes steps like merging the release branch into github, or creating release notes, publishing to NPM doesn't rely on those steps at all. 

You all can review the release process I proposed in `RELEASE_PROCESS.md`